src/execute/types.ts

```diff
@@ -18,31 +18,6 @@ export const soapBody = 'soapenv:Body';
 export const soapHeader = 'soapenv:Header';
 export const action = 'executeAnonymous';
 
-export const soapTemplate = `<env:Envelope xmlns:xsd="http://www.w3.org/2001/XMLSchema"
```

Comment:
used in 1 place, not really a "type".

turned it into a template literal for simplicity